build.sbt

```diff
@@ -4,7 +4,7 @@
 // Release tags should follow: http://semver.org/
 import scalariform.formatter.preferences._
 
-ThisBuild / git.baseVersion         := "1.3.0"
+ThisBuild / git.baseVersion         := "1.3.4"
```

Comment:
This is an unrelated change I guess, ideally not in this PR

Reply:
the current release is 1.3.3, so I don't know why this was set to 1.3.0.
I changed it to 1.3.4, so that my local publication is at a higher version than the release.
(perhaps it should still read 1.3.3 or 1.3.0?)